packages/logger/package.json

```diff
@@ -31,7 +31,7 @@
         "winston-transport": "4.5.0"
     },
     "devDependencies": {
-        "@types/debug": "4.1.8",
+        "@types/debug": "4.1.12",
```

Comment:
@ullasholla Do we really need this `devDependency`. I can't seem to find where it is used.